test/helper.rb

```diff
@@ -1,4 +1,15 @@
+require 'vcr'
+
 if ENV['TRAVIS']
   require 'coveralls'
   Coveralls.wear!
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('<AUTH_KEY>') { ENV['RENTLINX_AUTH_KEY'] }
+  config.filter_sensitive_data('<PASSWORD>') { ENV['RENTLINX_PASSWORD'] }
+  config.filter_sensitive_data('<USERNAME>') { ENV['RENTLINX_USERNAME'] }
```

Comment:
Could be, yes. Maybe one solution is for the yaml file to live in the user's config directory `HOME/.config/rentlinx_test.yml` and provide these default values for the case where that file doesn't exist. That only moves the storage from some shell source file, to another location, but it has a specific purpose.

Do you think it would be easier to document adding three environment variables, or adding three settings in a yaml file, and telling the user where to put it? Keep in mind the "default" location for config files varies by OS:
https://github.com/praw-dev/praw/blob/master/praw/settings.py#L31